hoodie-spark/src/main/scala/com/uber/hoodie/AvroConversionUtils.scala

```diff
@@ -272,12 +112,23 @@ object AvroConversionUtils {
       case ShortType => (item: Any) =>
         if (item == null) null else item.asInstanceOf[Short].intValue
       case _: DecimalType => (item: Any) => if (item == null) null else item.toString
```

Comment:
This line needs to be removed, otherwise `Decimal` is still being converted to `String`.